### ISSUE TITLE
Thousan values converts to decimal values on save

### ISSUE THREAD

**Issue by CMeldgaard** [verbb/gift-voucher#102]
### Describe the bug

When saving a voucher, that has a price, it sees the thousand seperator as decimal seperator and coverts for instance 5000 to just 5.

### Steps to reproduce

1. Input for instance 5000 
![image](https://user-images.githubusercontent.com/14794642/201860305-82e73303-ff22-415f-87b7-4d7d8b859ea8.png)

2. Save, and you get this, which is the 5000
![image](https://user-images.githubusercontent.com/14794642/201860424-b9bb37cc-f8ab-47a4-bfba-af32825a46f3.png)

3. Save it again, without changing anything
![image](https://user-images.githubusercontent.com/14794642/201860563-36b4a3ce-9688-4992-88c9-c36de74dbf0e.png)

Now it's saved as only 5

### Craft CMS version

4.3.1

### Plugin version

3.0.4

### Multi-site?

No

**Comment by engram-design** [comment]
Are you able to let me know what the locale of the currently-logged in user is (if different to the system local)? I know DKK is the Danish Krone, so just wanted to confirm your locale settings, which would have something to do with the internationalisation of these numbers.

**Comment by engram-design** [comment]
Actually, looks like this is happening across multiple locales.

Fixed for the next release. To get the fix early, change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-4 as 3.0.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by CMeldgaard** [comment]
Awesome :D Thank's @engram-design :D

**Comment by engram-design** [comment]
Fixed in [3.0.5](https://github.com/verbb/gift-voucher/releases/tag/3.0.5)
